### PR TITLE
[14.0][FIX] account_reconciliation_widget: in case of manual operation with…

### PR DESCRIPTION
**Pull request by jonerikceberio**
… analytic account, create the analytic entry if the move is posted

### CODE REVIEW

**Review by rp-landoo — APPROVED** [review]
Aprobación funcional

**Review by ao-landoo — APPROVED** [review]
Technical approval

**Comment by ao-landoo** [comment]
@pedrobaeza Could you review and merge if OK?

**Comment by pedrobaeza** [comment]
I don't get why they aren't automatically created when the move is posted.

File: account_reconciliation_widget/models/account_bank_statement.py [hunk 1]
```diff
@@ -256,6 +257,9 @@ def _create_counterpart_and_new_aml(
             self.move_id.with_context(
                 skip_account_move_synchronization=True
             ).action_post()
+        elif wo_aml.filtered(lambda l: l.analytic_account_id):
```
Comment (omar7r) [inline account_reconciliation_widget/models/account_bank_statement.py]: Please take into account analytic_tag_ids too, same problem

File: account_reconciliation_widget/models/account_bank_statement.py [hunk 2]
```diff
@@ -256,6 +257,9 @@ def _create_counterpart_and_new_aml(
...
+            # if already posted the analytic entry has to be created
+            wo_aml.filtered(lambda l: l.analytic_account_id).create_analytic_lines()
```
Comment (omar7r) [inline account_reconciliation_widget/models/account_bank_statement.py]: Please take into account analytic_tag_ids too, same problem

File: account_reconciliation_widget/models/account_bank_statement.py (hunk 1 above)
Comment (ao-landoo) [reply account_reconciliation_widget/models/account_bank_statement.py]: @jonerikceberio Could you make that change @omar7r is asking?

**Comment by marielejeune** [comment]
Hi @pedrobaeza 
I am not an account specialist, but when I try to reconcile a single item, the related move is already posted when entering into `process_reconciliation` and will not be reset to draft to be posted again. Hence the analytic line cannot be created without this fix.

**Comment by pedrobaeza** [comment]
OK, I understand what you say. But there are still some questions opened by Omar. Can you fix them?

**Comment by marielejeune** [comment]
If  @jonerikceberio doesn't answer within few days I'll try to do it

**Comment by jonerikceberio** [comment]
Tags taken into account, rebase and squash

**Comment by pedrobaeza** [comment]
@omar7r @ao-landoo can you confirm that everything is OK?

**Comment by ao-landoo** [comment]
> @omar7r @ao-landoo can you confirm that everything is OK?

Yes, it's OK

**Review by omar7r — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Can you please fw-port it to v15?

/ocabot merge patch

**Pull request merged**